src/TextUI/Configuration/TestSuiteBuilder.php

```diff
@@ -38,16 +42,27 @@ final class TestSuiteBuilder
     public function build(Configuration $configuration): TestSuite
     {
         if ($configuration->hasCliArgument()) {
-            $argument = realpath($configuration->cliArgument());
+            $arguments = [];
 
-            if (!$argument) {
-                throw new TestFileNotFoundException($configuration->cliArgument());
+            foreach (explode(' ', $configuration->cliArgument()) as $cliArgument) {
+                $argument = realpath($cliArgument);
+
+                if (!$argument) {
+                    throw new TestFileNotFoundException($cliArgument);
+                }
+
+                $arguments[] = $argument;
             }
 
-            $testSuite = $this->testSuiteFromPath(
-                $argument,
-                $configuration->testSuffixes(),
-            );
+            $testSuite = count($arguments) === 1
```

Comment:
Please do not use the ternary operator here.

Reply:
splitted using a more readable if-else.